controllers/reviews-controllers.js

```diff
@@ -0,0 +1,12 @@
+const { fetchReviews } = require("../models/reviews-models")
+
+exports.getReviews = (req, res, next) => {
+    const { restaurant_id } = req.params
+    fetchReviews(restaurant_id).then((review) => {
+        res.status(200).send({review})
+    })
+    .catch(err => {
```

Comment:
You are using promises, if your code has got to the catch block there is absolutely an error, adding an error callback here is redundant and should just be passed next. See section 3 of the error handling notes.